stable/opsportal/templates/hooks-roles.yaml

```diff
@@ -1,43 +1,61 @@
+# Creates a temporary administrative service account for the purpose of
+# running an elevated job that can create the ops-portal super user
 {{- if .Values.secrets.create }}
 ---
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+apiVersion: v1
+kind: ServiceAccount
 metadata:
-  name: {{ template "opsportal.fullname" . }}-secrets
+  name: "{{.Release.Name}}-service-account-pre-install"
```

Comment:
This runs a risk of being too long. Consider defining a template in _helpers.tpl to generate the name the same way fullname is generated.

Reply:
Good catch, I'll go back to using the truncated name in the template.

Reply:
For some reason, changing this to `{{ template "opsportal.fullname" . }}-admin` breaks the chart.

Reply:
I'll offer a patch when I get to a computer.